src/components/Detail/Menu/Category.tsx

```diff
@@ -0,0 +1,57 @@
+import { css, Theme } from "@emotion/react";
+
+function Category() {
+    return (
+        <>
+            <div css={keywordMenu}>
+                <span css={title}>메뉴</span>
+            </div>
+            <div css={categoryMenu}>
+                <div css={categoryShape}>
+                    <span css={category}>키워드</span>
+                </div>
+            </div>
+        </>
+    );
+}
+
+export default Category;
+
+const title = (theme: Theme) => css`
+  font-size: 1rem;
+  font-weight: ${theme.fontWeight.bold};
+`;
+
+const keywordMenu = (theme: Theme) => css`
+  width: 100%;
+  margin: 1.8rem 0 0.85rem 0;
+  padding: 0.55rem 1.3rem;
+  background-color: ${theme.color.grey100};
+`;
+
+const categoryMenu = css`
+  display: flex;
+  flex-direction: row;
+  padding: 0 0.7rem;
+  height: 2rem;
+  width: 100%;
+  overflow-x: scroll;
+
+  ::-webkit-scrollbar {
```

Comment:
```suggestion
  &::-webkit-scrollbar {
```

요런 형식으로 되어야 적용되지 않을까요?!

Reply:
허걱 맞아요 !!!